README.md

```diff
@@ -167,6 +167,30 @@ There are a few different areas in which we generate documentation content as a
     - We leverage [API-Extractor](https://api-extractor.com/) to generate summaries of our package APIs.
       This is done as a part of a full build, but it can also be executed in isolation by running `npm run build:api` from the repo root.
 
+## Common Workflows and Patterns
+
+This section contains common workflows and patterns to increase inner dev loop efficiency.
+
+### Build
+
+-   `pnpm install` from the root of the repository to install dependencies. This is necessary for new clones or after pulling changes from the main branch.
+-   `pnpm run build:fast` from the root of the repository to perform an incremental build that matches the CI build process. Incremental builds tend to leave extra files laying around, so running a clean is sometimes needed to cleanup ghost tests
+-   `pnpm run build:fast -- <path>` to build only a specific part of the repository.
+-   `pnpm run build` within a package directory to build that package.
+-   `pnpm run build:compile` for cross-package compilation.
+-   `pnpm run format` to format the code.
+-   `npx fluid-build -t build <NAME_OF_PACKAGES>` to build a single package or multiple space-separated packages along with all their dependencies.
```

Comment:
we shouldn't encourage `npx` in our common workflows. If we want to document this, we could ask people to `npm i -g @fluidframework/build-tools`, ensure their npm globally installed packages script location is in their PATH (or equivalent on other OSes), and drop the npx.

Note that this workflow is actually *only* useful for the multi-package setup. Single package build does the same thing as `pnpm run build` within a package directory (our build scripts use fluid-build with --task, which is what -t is short for)

Reply:
I have added a multi package setup category, please check it out and let me know what you think